homeassistant/components/scene/services.yaml

```diff
@@ -5,4 +5,10 @@ turn_on:
   fields:
     entity_id:
       description: Name(s) of scenes to turn on
-      example: 'scene.romantic'
+      example: "scene.romantic"
+
+reload:
+  description: Reload the scene configuration
+
+apply:
+  description: Activate a scene. Takes same data as the entities field from a single scene in the config.
```

Comment:
We probably want examples here too before release.

Reply:
So this is kinda the problem here. Each key is an entity ID. So there is no real "fields" defining. 

Wonder if the solution would be to add a "states" field and put all data under that ?